core/src/main/java/org/wso2/msf4j/internal/router/MicroserviceMetadata.java

```diff
@@ -181,7 +178,7 @@ private boolean isHttpMethodAvailable(Method method) {
                 throw new HandlerException(Response.Status.METHOD_NOT_ALLOWED, uri);
             } else {
                 throw new HandlerException(Response.Status.NOT_FOUND,
-                        String.format("Problem accessing: %s. Reason: Not Found", uri));
+                        String.format("Problem accessing: %s. Reason2: Not Found", uri));
```

Comment:
What would be the reason for changing "Reason" to "Reason2"?

Reply:
Oops, leftover from testing when I was trying to find out which of the two identical error messages was actually producing the client side message.  I will remove that.